### DISCUSSION TITLE
how to disable button tremble on click

### DISCUSSION THREAD

**Discussion by SteveSuv** [mantinedev/mantine#2413] [Q&A] [answered]
how to disable button tremble on click?

**Comment by rtivital** [comment, marked as answer]
Set custom active styles on theme – https://mantine.dev/theming/theme-object/#activestyles
In your case, styles should be empty

**Reply by Sajarin-M** [reply]
> Set custom active styles on theme – https://mantine.dev/theming/theme-object/#activestyles In your case, styles should be empty

How do I do it only for a single button?

**Reply by rtivital** [reply]
https://codesandbox.io/s/frosty-sun-bddu42?file=/src/App.tsx
